app/scripts/views/sign_up.js

```diff
@@ -48,14 +48,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     beforeRender: function () {
+      var self = this;
       if (document.cookie.indexOf('tooyoung') > -1) {
         this.navigate('cannot_create_account');
         return p(false);
       }
 
       this._bouncedEmail = this.ephemeralMessages.get('bouncedEmail');
-
-      return FormView.prototype.beforeRender.call(this);
+      require(['passwordcheck'], function (PasswordCheck) {
```

Comment:
In production this attaches all the password check code to the bundle.
### Before:
![image](https://cloud.githubusercontent.com/assets/128755/9063506/b980b98c-3a95-11e5-88f2-35372214d52b.png)
### After: 
![](http://v14d.com/i/55c0d05936cf3.png)

We probably do not want to do that but fetch script after the page loaded + only on desktop as @ckarlof mentioned a few weeks ago.